docs/bucket/notifications/README.md

```diff
@@ -392,7 +392,7 @@ $ curl  "http://localhost:9200/minio_events/_search?pretty=true"
               "source" : {
                 "host" : "127.0.0.1",
                 "port" : "38062",
-                "userAgent" : "Minio (linux; amd64) minio-go/2.0.3 mc/2017-02-15T17:57:25Z"
+                "userAgent" : "MinIO (linux; amd64) minio-go/2.0.3 mc/2017-02-15T17:57:25Z"
```

Comment:
This change also needs to be done in `minio-go`.

Reply:
Good reminder.
Will do when I work on minio-go.